.github/workflows/screener-run.yml

```diff
@@ -0,0 +1,198 @@
+            az storage blob upload-batch -d '$web/${{env.DEPLOYBASEPATH}}/react-northstar-screener' -s 'packages/fluentui/docs/dist' --overwrite
+
+      - name: Start @fluentui/react-northstar VR Test
+        run: yarn workspace @fluentui/docs vr:test
+        env:
+          SCREENER_ENDPOINT: ${{secrets.SCREENER_ENDPOINT}}
+          SCREENER_PROXY_ENDPOINT: ${{secrets.SCREENER_PROXY_ENDPOINT}}
+          SCREENER_API_KEY: ${{secrets.SCREENER_API_KEY}}
+
+      - name: 'check: modified files'
+        run: yarn check:modified-files
+
+  Screener:
+    runs-on: 'ubuntu-latest'
+
+    name: Screener @fluentui/react
+
+    steps:
+      - uses: actions/checkout@v3
+
+      - uses: actions/setup-node@v3
+        with:
+          node-version: 14.18.1
+
+      - name: Download artifact
+        uses: dawidd6/action-download-artifact@v2
+        with:
+          workflow: screener-build.yml
+          workflow_conclusion: success
+          name: env-artifact
+
+      - name: Define env variables
+        run: |
+          input_file="environment"
+          while read line
+          do
+          echo "$line" >> $GITHUB_ENV
+          done < "$input_file"
+
+      - name: Log environment variables (Linux)
+        if: runner.os == 'Linux'
+        run: |
+          printenv | sort ;\
+          echo "SHELLOPTS $SHELLOPTS" ;\
+
+      - name: Download screener storybook artifact
+        uses: dawidd6/action-download-artifact@v2
+        with:
+          workflow: screener-build.yml
+          workflow_conclusion: success
+          name: screener-artifact
+          path: apps/vr-tests/dist/storybook
+
+      - name: yarn
+        run: ./yarn-ci.sh
+        shell: bash
+
+      - name: Upload @fluentui/react VR test site
+        uses: azure/CLI@v1
+        with:
+          inlineScript: |
+            az storage blob upload-batch -d '$web/${{env.DEPLOYBASEPATH}}/react-screener' -s 'apps/vr-tests/dist/storybook' --overwrite
+
+      - name: Start @fluentui/react VR Test
+        run: yarn workspace @fluentui/vr-tests screener
+        env:
+          SCREENER_ENDPOINT: ${{secrets.SCREENER_ENDPOINT}}
+          SCREENER_PROXY_ENDPOINT: ${{secrets.SCREENER_PROXY_ENDPOINT}}
+          SCREENER_API_KEY: ${{secrets.SCREENER_API_KEY}}
+
+      - name: 'check: modified files'
+        run: yarn check:modified-files
+
+  ScreenerVNext:
+    runs-on: 'ubuntu-latest'
+
+    name: Screener @fluentui/react-components
+
+    steps:
+      - uses: actions/checkout@v3
+
+      - uses: actions/setup-node@v3
+        with:
+          node-version: 14.18.1
+
+      - name: Download artifact
+        uses: dawidd6/action-download-artifact@v2
+        with:
+          workflow: screener-build.yml
+          workflow_conclusion: success
+          name: env-artifact
+
+      - name: Define env variables
+        run: |
+          input_file="environment"
+          while read line
+          do
+          echo "$line" >> $GITHUB_ENV
+          done < "$input_file"
```

Comment:
> done < "$input_file"

What does this do ?

Reply:
It takes the downloaded artifact as input for the while loop so that each iteration takes the next line from that file